### PR TITLE
add map resizable , add current work dir

### PR DESCRIPTION
**Pull request by sunqirui1987**

### CODE REVIEW

File: fs/fsutil/dir.go [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+func GetCurrDir() string {
+	dir := getCurrentAbPathByExecutable()
+	if strings.Contains(dir, os.TempDir()) {
+		dir, _ = os.Getwd()
+	}
+	return dir
+}
+
+// 获取当前执行程序所在的绝对路径
```
Comment (xushiwei) [inline fs/fsutil/dir.go:18]: don't have any chinese.

File: game.go [hunk 2]
```diff
@@ -310,7 +314,8 @@ type cameraConfig struct {
 }
 
 type mapConfig struct {
-	Mode string `json:"mode"`
+	Mode      string `json:"mode"`
+	Resizable bool   `json:"resizable"`
```
Comment (xushiwei) [inline game.go:318]: resizable is a window property, not a map property.

File: fs/fsutil/dir.go [hunk 3]
```diff
@@ -0,0 +1,26 @@
+package fsutil
+
+import (
+	"log"
+	"os"
+	"path/filepath"
+	"strings"
+)
+
+func GetCurrDir() string {
```
Comment (xushiwei) [inline fs/fsutil/dir.go:10]: this should be a private function, because `fs/fsutil` isn't a private package.

File: game.go [hunk 4]
```diff
@@ -120,6 +122,8 @@ func (p *Game) initGame() {
 
 // Gopt_Game_Main is required by Go+ compiler as the entry of a .gmx project.
 func Gopt_Game_Main(game Gamer) {
+	dir := fsutil.GetCurrDir()
+	os.Chdir(dir)
```
Comment (xushiwei) [inline game.go:126]: this means we can't use `gop run .` to start a spx game?

File: game.go [hunk 5]
```diff
@@ -702,9 +708,11 @@ func (p *Game) runLoop(cfg *Config) (err error) {
 }
 
 func (p *Game) Layout(outsideWidth, outsideHeight int) (screenWidth, screenHeight int) {
-	p.windowWidth_ = outsideWidth
-	p.windowHeight_ = outsideHeight
-	p.resizeWindow()
+	if p.resizable == true {
```
Comment (xushiwei) [inline game.go:711]: don't need `== true`.

File: fs/fsutil/dir.go (hunk 1 above)
Comment (sunqirui1987) [reply fs/fsutil/dir.go]: fixed

File: game.go (hunk 2 above)
Comment (sunqirui1987) [reply game.go]: fixed

File: game.go (hunk 5 above)
Comment (sunqirui1987) [reply game.go]: fixed

**Pull request closed without merging**